src/status_im/chat/views/input/input.cljs

```diff
@@ -103,13 +103,13 @@
         :placeholder-text-color style/color-input-helper-placeholder
         :auto-capitalize        :sentences}])))
 
-(defn- invisible-input [{:keys [set-layout-width-fn value]}]
+(defn- invisible-input [{:keys [set-layout-height-fn container-width]}]
   (let [input-text    (subscribe [:chat :input-text])]
-    [text {:style     style/invisible-input-text
-           :on-layout #(let [w (-> (.-nativeEvent %)
+    [text {:style     (style/invisible-input-text container-width)
+           :on-layout #(let [h (-> (.-nativeEvent %)
                                    (.-layout)
-                                   (.-width))]
-                         (set-layout-width-fn w))}
```

Comment:
We change from `height` to `width`? Any special reason?

Reply:
Width used to be updated but not anymore? Do we expect that?

Reply:
it is updated by 'basic-text-input' now.

Reply:
So this code wasn't necessary?

Reply:
I tried to delete it :) no effect...

Reply:
it may be just historical - see 'value' parameter ?